packages/derived-wallet-ethereum/src/createSiweEnvelope.ts

```diff
@@ -20,7 +20,7 @@ function createSiweEnvelope(input: CreateSiweEnvelopeInput & { statement: string
   return createSiweMessage({
     address: ethereumAddress,
     domain: window.location.host,
-    uri: window.location.host,
+    uri: window.location.protocol + '//' + window.location.host,
```

Comment:
we can just `window.location.origin` here.

Maybe the better thing to do would be to have `domain` and `scheme` passed separately as inputs so that the message is built from its components, just like on chain.

But not needed now and outside of the scope of this PR probably